dev-lang/rust/rust-1.58.1.ebuild

```diff
@@ -105,7 +105,11 @@ DEPEND="
 	net-misc/curl:=[http2,ssl]
 	sys-libs/zlib:=
 	dev-libs/openssl:0=
-	elibc_musl? ( sys-libs/libunwind:= )
+	elibc_musl? (
+		llvm-libunwind? ( sys-libs/llvm-libunwind:= )
+		!llvm-libunwind? ( sys-libs/libunwind:= )
+	)
```

Comment:
```
llvm-libunwind? ( sys-libs/llvm-libunwind:= )
!llvm-libunwind? ( elibc_musl?  ( sys-libs/libunwind:= ) )
```
this is what you probably want. may be also complicated by system-llvm, not sure yet.

Reply:
This would change behavior on glibc systems, which is something I'd rather not do.